src/game_interpreter.cpp

```diff
@@ -647,9 +647,9 @@ bool Game_Interpreter::ExecuteCommand(lcf::rpg::EventCommand const& com) {
 		case Cmd::ChangeHeroTitle:
 			return CommandChangeHeroTitle(com);
 		case Cmd::ChangeSpriteAssociation:
-			return CommandChangeSpriteAssociation(com);
+			return CmdSetup<&Game_Interpreter::CommandChangeSpriteAssociation, 3>(com); // FIX OOB access
 		case Cmd::ChangeActorFace:
-			return CommandChangeActorFace(com);
+			return CmdSetup<&Game_Interpreter::CommandChangeActorFace, 2>(com); // FIX OOB access
```

Comment:
Don't think the "Fix OOB access" comment is necessary, this function call will be slowly added to all commands.